snakeplane/constants.py

```diff
@@ -0,0 +1 @@
+SNAKEPLANE_NULL_VALUE_PLACEHOLDER = "sNaKePlAnE_nUlL_vAlUe"
```

Comment:
Could also just use a UUID, here's a V4 UUID from [here](https://www.uuidgenerator.net/version4):
`0c2856d0-f631-445a-a112-33fa282cb5de`

Reply:
Not a big deal either way though.

Reply:
we'd want this to be constant, and not change.  I'm ok with a UUID as long as it's a constant, and not randomly generated each time.

Reply:
Yes that's what I mean. Wasn't clear though sorry.

Reply:
You guys are so lame.

Reply:
<a href="https://imgflip.com/i/3h9efv"><img src="https://i.imgflip.com/3h9efv.jpg" title="made at imgflip.com"/></a>

Reply:
![image](https://user-images.githubusercontent.com/12815528/69465075-6496ec00-0d3d-11ea-9f85-aec3aea927ce.png)